tests/test_examples.py

```diff
@@ -1313,3 +1316,112 @@ def test_optimize_with_fourier_planar_coil():
     optimizer = Optimizer("fmintr")
     (c,), _ = optimizer.optimize(c, objective=objective, maxiter=200, ftol=0, xtol=0)
     np.testing.assert_allclose(c.compute("length")[1]["length"], 11, atol=1e-3)
+
+
+@pytest.mark.unit
+@pytest.mark.slow
```

Comment:
This takes a minute or so to run on my laptop, since `VMECIO.save` is slow. If we really need to speed this up we could either reduce the equilibrium resolution or manually save only the VMEC quantities that are used.

Reply:
I sped this up significantly by manually saving the few quantities instead of calling VMECIO.save